src/main.rs

```diff
@@ -81,8 +81,12 @@ fn main() -> Result<(), Box<dyn Error>> {
         );
     let matches = match opts.parse(std::env::args()) {
         Ok(it) => it,
+        Err(getopts::Fail::UnrecognizedOption(string)) => {
+            eprintln!("{}", opts.usage(&format!("Unknown option '{string}'.")));
+            // Quit the application with `EX_USAGE` from `sysexits.h`.
+            std::process::exit(64);
+        }
         Err(err) => {
-            commit_analyzer::usage(opts);
```

Comment:
I was just about to merge this, but then I checked the possible Err(_) values, this can return:

[reference](https://docs.rs/getopts/latest/getopts/enum.Fail.html)

I think, that the usage should still be printed, if the user has its arguments in an invalid way.

Reply:
Okay, I will change this.  By the way, here is the sysexits repository including examples:  https://github.com/sorairolake/sysexits-rs.  I will incorporate this idea.

Reply:
:wrench: :heavy_check_mark: